dist/index.html

```diff
@@ -43,6 +37,6 @@ <h2>Add your score</h2>
             </div>
         </section>
     </div>
-    <script src="./index.bundle.js"></script>
+    <script src="main.js"></script>
```

Comment:
functionalities are not working because the javascript is not present in the index.html file 😸  please change the directory name to `index.js`

Reply:
i have fixed that. when you run npm run build, it will build your files then run npm start to run your  webpack main project